libs/langchain-ollama/src/chat_models.ts

```diff
@@ -56,6 +56,11 @@ export interface ChatOllamaCallOptions extends BaseChatModelCallOptions {
   tools?: BindToolsInput[];
   // eslint-disable-next-line @typescript-eslint/no-explicit-any
   format?: string | Record<string, any>;
+  /**
+   * Whether to enable thinking mode for this specific invocation.
+   * Can be a boolean (true/false) or a string intensity level ("high", "medium", "low").
+   */
+  think?: boolean | 'high' | 'medium' | 'low';
```

Comment:
It seems we are using this type 3 times across different places, should we create a type for this and reference instead?